src/main/java/com/limechain/runtime/hostapi/OffchainHostFunctions.java

```diff
@@ -5,31 +5,43 @@
 import com.limechain.network.Network;
 import com.limechain.network.protocol.blockannounce.NodeRole;
 import com.limechain.rpc.server.AppBean;
+import com.limechain.runtime.hostapi.dto.HttpErrorType;
+import com.limechain.runtime.hostapi.dto.HttpStatusCode;
 import com.limechain.runtime.hostapi.dto.InvalidArgumentException;
+import com.limechain.runtime.hostapi.dto.InvalidRequestId;
 import com.limechain.runtime.hostapi.dto.RuntimePointerSize;
+import com.limechain.runtime.hostapi.exceptions.OffchainResponseWaitException;
 import com.limechain.storage.KVRepository;
 import com.limechain.storage.offchain.OffchainStore;
 import com.limechain.sync.warpsync.SyncedState;
+import com.limechain.utils.scale.ScaleUtils;
 import com.limechain.utils.scale.exceptions.ScaleEncodingException;
 import io.emeraldpay.polkaj.scale.ScaleCodecReader;
 import io.emeraldpay.polkaj.scale.ScaleCodecWriter;
+import io.emeraldpay.polkaj.scale.reader.ListReader;
+import io.emeraldpay.polkaj.scale.reader.UInt64Reader;
+import io.emeraldpay.polkaj.scale.writer.ListWriter;
 import io.emeraldpay.polkaj.scaletypes.Result;
 import io.emeraldpay.polkaj.scaletypes.ResultWriter;
 import io.libp2p.core.PeerId;
 import io.libp2p.core.multiformats.Multiaddr;
+import kotlin.Pair;
```

Comment:
Since `kotlin` is a transitive dependency, we had earlier agreed to use `org.javatuples.Pair` instead. Just for consistency.

Reply:
The kotlin pair is used in `ScaleUtils.Encode.encode`, which we call in this class.

Reply:
Okay, sorry, disregard. Will be refactored later.